soql-sqlizer/src/test/scala/com/socrata/soql/sqlizer/SqlizerTest.scala

```diff
@@ -297,4 +297,13 @@ class SqlizerTest extends FunSuite with MustMatchers with TestHelper with Sqlize
     val sqlish = analyze(tf, soql).layoutSingleLine.toString
     sqlish must equal ("SELECT 1.0 :: numeric AS i1, 2.0 :: numeric AS i2")
   }
+
+  test("order by a selected function call which produces a non-compressed compound column") {
+    val tf = tableFinder()
+    val soql = "select expanded_compound() as ec from @single_row order by ec"
+    val sqlish = analyze(tf, soql, Seq(rewrite.Pass.UseSelectListReferences)).layoutSingleLine.toString
+
+    // and in particular _not_ "select "c a", "c b" order by compress(1, 2)"...
+    sqlish must equal ("""SELECT "column a" AS i1_a, "column b" AS i1_b ORDER BY test_soql_compress_compound("column a", "column b") ASC NULLS LAST""")
```

Comment:
NGL, I read these and thought they were actual columns for a moment before I realized they were just literals.

Reply:
Yeah, the dialect of SQL that the test-sqlizer produces uses strings that are quoted like JSON.